src/gui/mainwindow.ui

```diff
@@ -95,12 +95,23 @@
     <property name="title">
      <string>&amp;View</string>
     </property>
+    <widget class="QMenu" name="menu_Log">
```

Comment:
Please use the same rule for names as in c++ files. ```menuLog```, ```actionExecutionLogs```, etc.
Existing names in MainWindow I fixed in one of my last PRs.